views/apartments/index.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>App</title>
+</head>
+<body>
+  <h1>Apartments</h1>
+  <% @apartment.each do |apartment| %>
+  <!-- <h2>Delete Apartment</h2> -->
+
+  <p class="apartment">
+    <a href="/apartments/<%= apartment.id %>" >
+      <body style="height: 4em" src= "<%= artist.photo_url %>" >
```

Comment:
not sure how `artist.photo_url` got into your code, but this looks like something from Tunr and not landlord.